docs/static/generated/rest_v1_dispatcher.yml

```diff
@@ -2663,6 +2679,19 @@ components:
       type: object
       allOf:
       - $ref: '#/components/schemas/SubtaskCheckpointStatistics'
+    Plan:
+      type: object
+      properties:
+        jid:
+          type: string
+        name:
+          type: string
+        nodes:
+          type: array
+          items:
+            $ref: '#/components/schemas/Node'
```

Comment:
Is this type reference correct? 🤔 

It looks like `#/components/schemas/Node` refers to `org.apache.flink.runtime.webmonitor.threadinfo.VertexFlameGraph.Node`, but it should refer to 
`org.apache.flink.runtime.rest.messages.JobPlanInfo.Plan.Node`

Reply:
Renamed the schema to `PlanNode`